src/Custom-sample.h

```diff
@@ -110,26 +110,26 @@
 #define DEFAULT_PIN_RESET_BUTTON                (-1)
 
 
-#define DEFAULT_USE_RULES                       false             // (true|false) Enable Rules?
+#define DEFAULT_USE_RULES                       true              // (true|false) Enable Rules?
 #define DEFAULT_RULES_OLDENGINE                 true
 
-#define DEFAULT_MQTT_RETAIN                     false             // (true|false) Retain MQTT messages?
+#define DEFAULT_MQTT_RETAIN                     true              // (true|false) Retain MQTT messages?
 #define DEFAULT_MQTT_DELAY                      100               // Time in milliseconds to retain MQTT messages
 #define DEFAULT_MQTT_LWT_TOPIC                  ""                // Default lwt topic
 #define DEFAULT_MQTT_LWT_CONNECT_MESSAGE        "Connected"       // Default lwt message
 #define DEFAULT_MQTT_LWT_DISCONNECT_MESSAGE     "Connection Lost" // Default lwt message
 #define DEFAULT_MQTT_USE_UNITNAME_AS_CLIENTID   0
 
-#define DEFAULT_USE_NTP                         false             // (true|false) Use NTP Server
-#define DEFAULT_NTP_HOST                        ""                // NTP Server Hostname
-#define DEFAULT_TIME_ZONE                       0                 // Time Offset (in minutes)
-#define DEFAULT_USE_DST                         false             // (true|false) Use Daily Time Saving
+#define DEFAULT_USE_NTP                         true              // (true|false) Use NTP Server
+#define DEFAULT_NTP_HOST                        "pool.ntp.org"    // NTP Server Hostname
```

Comment:
There is no need to give this DNS for NTP as that's the default when no host is given.